Meditation/Meditation/Model/MoodsData.swift

```diff
@@ -2,5 +2,10 @@ import SwiftUI
 
 class MoodsData: ObservableObject {
     @Published var selectedMoodIndex = 0
-    var moods = ["Sweet Sleep", "Insomnia", "Depressic", "Disorder"]
+    var moods = [
+        NSLocalizedString("Sweet Sleep", comment: ""),
```

Comment:
Check out the `localized()` extension in CW.

Reply:
Yes, I saw, but Vlad and Alla said that it is not necessary to do this(maybe at this stage :) ) and I don't understand what is the point of that extension.